docs/source/guides/db_integration_with_featuretools_sql.ipynb

```diff
@@ -0,0 +1,125 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What is `featuretools_sql`? \n",
+    "\n",
+    "`featuretools_sql` is an add-on library to `featuretools` that supports automatic `EntitySet` creation from relational data stored in a relational database management system (RDBMS).\n",
+    "\n",
+    "Currently, `featuretools_sql` is compatible with the following RDBMS's: `MySQL` and `PostgreSQL`. \n",
```

Comment:
- List the supported RDBMs in a indent list (because we will add support for more in the future)